substrate/frame/system/src/offchain.rs

```diff
@@ -489,9 +483,37 @@ pub trait CreateSignedTransaction<LocalCall>:
 }
 
 /// Interface for creating an inherent.
+///
+/// Implement this trait for the runtime but use `CreateBare` instead.
+/// This trait is defined to avoid a breaking change, and it automatically implements `CreateBare`.
 pub trait CreateInherent<LocalCall>: CreateTransactionBase<LocalCall> {
```

Comment:
```suggestion
#[deprecated(note = "Please use `CreateBare` instead")]
pub trait CreateInherent<LocalCall>: CreateTransactionBase<LocalCall> {
```

Reply:
Then I will also change usage of `CreateInherent` to `CreateBare` in pokadot-sdk, in pallets and runtime

Reply:
> EDIT: otherwise runtime implementing only `CreateBare` won't be able to use like `election-solution` pallet which uses `CreateInherent`.

I mean you can still implement `CreateInherent`, just with a `allow(deprecated)`.